stdlib/http/src/main/ballerina/src/http/cookie/cookieStore.bal

```diff
@@ -14,14 +14,21 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/log;
 import ballerina/time;
 
 # Represents the cookie store.
 #
 # + allSessionCookies - Array to store all the session cookies
+# + persistentCookieHandler - Persistent cookie handler to manage persistent cookies
 public type CookieStore object {
 
     Cookie[] allSessionCookies = [];
+    PersistentCookieHandler? persistentCookieHandler = ();
+
+    public function __init(PersistentCookieHandler? persistentCookieHandler) {
```

Comment:
Shall we add a default value for the `persistentCookieHandler` value ?
Like,

`public function __init(PersistentCookieHandler? persistentCookieHandler = ())` ?

This way we can instantiate a CookieStore by 

`CookieStore cookieStore = new;`
instead of 
`CookieStore cookieStore = new(());` which is kind of weird (for me)

WDYT ?